packages/compartment-mapper/src/policy.js

```diff
@@ -459,61 +438,3 @@ export const attenuateModuleHook = async (
     originalModuleRecord,
   });
 };
-
-const padDiagnosis = text => ` (${text})`;
```

Comment:
With compartments for disallowed dependencies being defined in the compartment map, loading them will reach the point where policy is checked and we now can avoid all the guesswork below.